operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -1361,9 +1351,7 @@ func (rc *ReconciliationContext) CheckDcPodDisruptionBudget() result.ReconcileRe
 	return result.Continue()
 }
 
-// UpdateRackNodeCount ...
```

Comment:
don't delete this - make a better docstring - the standards/idioms/linter really wants exported stuff to have a docstring